.circleci/config.yml

```diff
@@ -218,6 +223,37 @@ jobs:
             . ./venv/Scripts/activate
             pytest nevergrad -v
 
+  pypi-deploy:
+    docker:
+      - image: python:3.6
+    steps:
+
+      - checkout
+
+      - attach_workspace:
+          at: nevergrad-build
+
+      - run:
+          name: init .pypirc
+          command: |
+            echo -e "[pypi]" >> ~/.pypirc
+            echo -e "username = jrapin" >> ~/.pypirc
+            echo -e "password = $PYPI_PASSWORD" >> ~/.pypirc
+
+      - run:
+          name: Verify tag
+          command: |
+            python setup.py verify
+
+      - run:
+          name: upload to pypi
+          command: |
+            python3 -m venv_deploy venv
```

Comment:
Did you mean:
```suggestion
            python3 -m venv venv_deploy
```

Reply:
Yes I figured as well indeed :D thanks ;)